tests/core/snap-set-core-config/task.yaml

```diff
@@ -92,42 +92,28 @@ execute: |
         exit 1
     fi
 
-
-    rm -f /var/lib/console-conf/complete
-    echo "Check that 'service.console-conf.disable' handling works"
-    # start some instances of console-conf
-    systemctl start console-conf@tty1
-    systemctl start console-conf@tty2
-    systemctl start serial-console-conf@ttyS0
-
-    systemctl status 'console-conf@*' | MATCH "Active: active"
-    systemctl status 'serial-console-conf@*' | MATCH "Active: active"
-
-    snap set system service.console-conf.disable=true
-    test -e /var/lib/console-conf/complete
-    # no output for anything console-conf
-    test -z "$(systemctl status 'console-conf@*')"
-    # serial-console-conf is usually in failed status, depending on how the
-    # process went down
-    systemctl status 'serial-console-conf@*' | MATCH "(|Active: (inactive|failed))"
-
-    # getty@ services are now active
-    systemctl status getty@tty1 | MATCH "Active: active"
-    systemctl status getty@tty2 | MATCH "Active: active"
-    systemctl status serial-getty@ttyS0 | MATCH "Active: active"
-
-    # enable it again
-    snap set system service.console-conf.disable=false
-    not test -e /var/lib/console-conf/complete
-
-    # console-conf@ services are now active
-    systemctl status 'console-conf@tty1' | MATCH "Active: active"
-    systemctl status 'console-conf@tty2' | MATCH "Active: active"
-    systemctl status 'serial-console-conf@ttyS0' | MATCH "Active: active"
-    # but getty@ ones are not
-    systemctl status getty@tty1 | MATCH "Active: (inactive|failed)"
-    systemctl status getty@tty2 | MATCH "Active: (inactive|failed)"
-    systemctl status serial-getty@ttyS0 | MATCH "Active: (inactive|failed)"
+    echo "Changing 'service.console-conf.disable' at runtime does not work"
+    console_conf_disabled="false"
+    if [ -e /var/lib/console-conf/complete ]; then
+        console_conf_disabled="true"
```

Comment:
Probably a silly question, but why is this conditional? Isn't the state of console-conf known upfront as test precondition? If this depends on core versions, shouldn't we make it conditional based on that?

Reply:
It's a good question - I did it this way to not be robust against different system configurations, i.e. if a testsystem has console-conf disabled this test works and if not too. But I guess I could also write /var/lib/console-conf complete myself in the test. I am not super sure if this may have bad side-effects hence I did it this way.

Reply:
I see, I think that's ok and not a blocker. Would be nice to have predefined test conditions to know that both of these cases are tested, but it can be a followup if you think it's worth pursuing.

Reply:
Thanks, I think followup is fine. I need to mediate a bit over this, I'm a bit unsure about messing around with /var/lib/console-conf/complete on the testsystem but after thinking it's probably okay.